dali/pipeline/operators/reader/loader/video_loader.h

```diff
@@ -119,14 +122,14 @@ class VideoLoader : public Loader<GPUBackend, SequenceWrapper> {
      */
     av_register_all();
 
-    // TODO(spanev) Implem several files handling
-    total_frame_count_ = get_or_open_file(filenames_[0]).frame_count_;
+    total_frame_count_ = 0;
 
-    // TODO(spanev) to change after deciding what we want
-    int seq_per_epoch = total_frame_count_ / count_;
-    frame_starts_.resize(seq_per_epoch);
-    for (int i = 0; i < seq_per_epoch; ++i) {
-      frame_starts_[i] = i * count_;
+    for (size_t i = 0; i < filenames_.size(); ++i) {
+      int frame_count = get_or_open_file(filenames_[i]).frame_count_;
+      for (int s = 0; s < frame_count; s += step_) {
+        frame_starts_.push_back(std::make_pair(i, s));
+      }
+      total_frame_count_ += frame_count;
```

Comment:
total_frame_count_ is used to define epoch size, so it should tell how many sequences, in total, we are able to get form provided input files.
Isn't that just `frame_starts_.size()`?